src/gar.lua

```diff
@@ -2233,4 +2233,37 @@ function gar.getPaginatedDelegatesFromAllGateways(cursor, limit, sortBy, sortOrd
 	)
 end
 
+--- @param cursor string|nil -- cursorId of the last item in the previous page
+--- @param limit number
+--- @param sortBy string|nil
```

Comment:
Do the type docs let us narrow the valid strings down?

Reply:
aaf5183